data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg

```diff
@@ -158,51 +160,51 @@
             {TRAIT_HEALTHY}
         [/modifications]
 
-        hitpoints=1
+        {QUANTITY hitpoints 3 2 1}
```

Comment:
What's the difference between 1hp and 3hp? Both of them sound like a one-hit kill, and even the hp was raised to taking more than one hit then that would still sound like a design bug - if he's expected to survive any hits at all then there's a reliance on the RNG.

Reply:
maybe increase the variance a bit more, then?

Reply:
I thought the purpose of it was to make the player keep him at the back for this scenario, but it's a long time since I played it.

Reply:
> I thought the purpose of it was to make the player keep him at the back for this scenario, but it's a long time since I played it.

Well, he's AI-controlled, and the AI retreats him anyways (has him switching between the keep for recruiting, and the nearest village for healing), so...

Reply:
Also, if he manages to survive the scenario, he's added to your recall list and gets auto-recalled in future scenarios, so there's an incentive to try to keep him alive

Reply:
Ah, if there's an incentive to keep him alive, then this seems fine to me.

Reply:
> Ah, if there's an incentive to keep him alive, then this seems fine to me.

should I increase the variance at all, then? Or just keep it as-is?

Reply:
Well, what's the aim of the change, and does it do that? How would a larger variance change the player's options?

The reason that I asked about this is that it seemed to be just finding some numbers and making them larger, rather than analysing the gameplay and then finding places to tweak how it responds to a lower skill level.

Reply:
> Well, what's the aim of the change, and does it do that? How would a larger variance change the player's options?
> 
> The reason that I asked about this is that it seemed to be just finding some numbers and making them larger, rather than analysing the gameplay and then finding places to tweak how it responds to a lower skill level.

The aim of the change is to make it easier to keep Stalrag alive on easier difficulty levels, which is something I had difficulty doing when I played through it. Yes, in some sense it is just finding numbers and making them larger, but I still think doing even that is worth it, though, as it adds replay value when things are different between difficulty levels, as it'll inspire players to try all of them to see what got changed.

Reply:
Having the aim is good, but the rest of your answer seems to be missing my point. You're jumping from the aim to changing numbers instead of analysing what change to make.

With that aim, maybe the analysis would be "on easy he should always survive one troll getting through his defenses", and then the numbers come from how much a strong troll could do to him in a single turn.

Reply:
I think #8337 might be relevant here

Reply:
I agree that 3/2/1 is not a reasonable change since it's just too minor. I'd suggest just going up at least slightly in some way (e.g. healing him after capturing the lich via event or something)

Reply:
> With that aim, maybe the analysis would be "on easy he should always survive one troll getting through his defenses", and then the numbers come from how much a strong troll could do to him in a single turn.

Well if I went with a troll, that would be a rather drastic variance... how about if instead we use the weakest unit that's pre-placed for side 3, the Young Ogre. 1 hit from a Young Ogre is 5 damage. So, `{11 6 1}`, then?